hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -67,6 +67,9 @@ def __init__(self, **kwargs: Any) -> None:
         default_kwargs = dict(
             # declared in BaseMILTiles:
             is_caching=False,
+            max_bag_size=56,
+            max_bag_size_inf=0,
+            batch_size=8,
```

Comment:
Small suggestion: maybe declare this in BaseDeepSMILEPanda that works for both tiles and slides?

Reply:
Good point. It is already declared in `DeepSMILESlidesPanda`, could declare it only once in `BaseDeepSMILEPanda`

Reply:
done, thanks